gfx/UltimateCataclysm/pngs_normal_32x32/vehicle/scooter/scooter_wheels.json

```
@@ -0,0 +1,14 @@
+[
+  {
+    "id": [ "vp_wheel_small_scooter", "vp_wheel_small_scooter_steerable" ],
+    "fg": [ "vp_wheel_scooter_rotN", "vp_wheel_scooter_rotW", "vp_wheel_scooter_rotS", "vp_wheel_scooter_rotE" ],
+    "rotates": true,
+    "bg": ""
+  },
+  {
+    "id": [ "vp_wheel_small_scooter_rear" ],
```

```suggestion
    "id": "vp_wheel_small_scooter_rear",
```
this can be simplified to be just a string